### DISCUSSION TITLE
Micropython To PC Display

### DISCUSSION THREAD

**Discussion by zcannon78** [peterhinch/micropython-nano-gui#32] [General]
Hello Peter,

I am a intermediate python programmer who wants to start using micropython.  I was wondering if their is a module that would allow me to display a gui (on my computer screen) that interacts with the pyboard or a similar microcontroller via the usb port. Something similar to Tkinter and Pyside.

**Comment by peterhinch** [comment]
I don't know of one, but I suggest you ask in [the forum](https://forum.micropython.org/index.php).
